src/cascadia/TerminalApp/defaults.json

```diff
@@ -200,41 +200,51 @@
-        { "command": "openSettings", "keys": ["ctrl+,"] },
-        { "command": "paste", "keys": ["ctrl+shift+v"] },
-        { "command": "prevTab", "keys": ["ctrl+shift+tab"] },
-        { "command": "scrollDown", "keys": ["ctrl+shift+down"] },
-        { "command": "scrollDownPage", "keys": ["ctrl+shift+pgdn"] },
-        { "command": "scrollUp", "keys": ["ctrl+shift+up"] },
-        { "command": "scrollUpPage", "keys": ["ctrl+shift+pgup"] },
-        { "command": { "action": "switchToTab", "index": 0 }, "keys": ["ctrl+alt+1"] },
-        { "command": { "action": "switchToTab", "index": 1 }, "keys": ["ctrl+alt+2"] },
-        { "command": { "action": "switchToTab", "index": 2 }, "keys": ["ctrl+alt+3"] },
-        { "command": { "action": "switchToTab", "index": 3 }, "keys": ["ctrl+alt+4"] },
-        { "command": { "action": "switchToTab", "index": 4 }, "keys": ["ctrl+alt+5"] },
-        { "command": { "action": "switchToTab", "index": 5 }, "keys": ["ctrl+alt+6"] },
-        { "command": { "action": "switchToTab", "index": 6 }, "keys": ["ctrl+alt+7"] },
-        { "command": { "action": "switchToTab", "index": 7 }, "keys": ["ctrl+alt+8"] },
-        { "command": { "action": "switchToTab", "index": 8 }, "keys": ["ctrl+alt+9"] },
-        { "command": "decreaseFontSize", "keys": ["ctrl+-"] },
-        { "command": "increaseFontSize", "keys": ["ctrl+="] },
-        { "command": "toggleFullscreen", "keys": ["alt+enter"] },
-        { "command": "toggleFullscreen", "keys": ["f11"] }
+      { "command": "closePane", "keys": [ "ctrl+shift+w" ] },
+      { "command": "closeWindow", "keys": [ "alt+f4" ] },
+      { "command": "copy", "keys": [ "ctrl+shift+c" ] },
+      { "command": "decreaseFontSize", "keys": [ "ctrl+-" ] },
+      { "command": "duplicateTab", "keys": [ "ctrl+shift+d" ] },
+      { "command": "increaseFontSize", "keys": [ "ctrl+=" ] },
+      { "command": { "action": "moveFocus", "direction": "down" }, "keys": [ "alt+down" ] },
+      { "command": { "action": "moveFocus", "direction": "left" }, "keys": [ "alt+left" ] },
+      { "command": { "action": "moveFocus", "direction": "right" }, "keys": [ "alt+right" ] },
+      { "command": { "action": "moveFocus", "direction": "up" }, "keys": [ "alt+up" ] },
+      { "command": "newTab", "keys": [ "ctrl+shift+t" ] },
+      { "command": "newTabProfile0", "keys": [ "ctrl+shift+1" ] },
+      { "command": "newTabProfile1", "keys": [ "ctrl+shift+2" ] },
+      { "command": "newTabProfile2", "keys": [ "ctrl+shift+3" ] },
+      { "command": "newTabProfile3", "keys": [ "ctrl+shift+4" ] },
+      { "command": "newTabProfile4", "keys": [ "ctrl+shift+5" ] },
+      { "command": "newTabProfile5", "keys": [ "ctrl+shift+6" ] },
+      { "command": "newTabProfile6", "keys": [ "ctrl+shift+7" ] },
+      { "command": "newTabProfile7", "keys": [ "ctrl+shift+8" ] },
+      { "command": "newTabProfile8", "keys": [ "ctrl+shift+9" ] },

```

Comment:
why does this revert these to the legacy key bindings without args..?

Reply:
Oh that's my bad, didn't notice the change in master